lib/file-helper.js

```diff
@@ -98,6 +98,25 @@ const getComponentsData = async () => {
   return Promise.all(componentNames.map(getComponentData))
 }
 
+/**
+ * Get examples from a component's metadata file
+ *
+ * @param {string} componentName - Component name
+ * @returns {Promise<Object<string, ComponentExample['data']>>} returns object that includes all examples at once
+ */
+async function getExamples (componentName) {
```

Comment:
Not sure why this one wasn't alongside the other helpers like `getFullPageExamples()`